src/Countries/Country.php

```diff
@@ -41,6 +41,41 @@ public function get(int $year, ?string $locale = null): array
         return $translatedHolidays;
     }
 
+    /** @return array<string, string>  date => name */
+    public function getInRange(?CarbonImmutable $from, ?CarbonImmutable $to): array
```

Comment:
@inDeev IMO, the parameters should not be nullable. I can't think of a use case when someone might want to use this method and not pass both of the range parameters.
Accepting string parameters also might be useful. 
```suggestion
    public function getInRange(CarbonImmutable|string $from, CarbonImmutable|string $to): array
```